packages/utils/src/utils/index.js

```diff
@@ -116,9 +116,7 @@ export function getStatus(resource) {
 }
 
 export function isRunning(reason, status) {
-  return (
-    status === 'Unknown' && (reason === 'Running' || reason === 'Building')
-  );
+  return status === 'Unknown' && reason === 'Running';
```

Comment:
Definitely safe to do? What was `Building` ever used for/set? I don't see a reference to it being removed here

Reply:
Building was what used before it changed to Run/Running
All instances of Building being used were removed before this PR